src/persist/src/indexed/trace.rs

```diff
@@ -40,10 +48,12 @@ pub struct BlobTrace<K, V> {
     id: Id,
     // The next ID used to assign a Blob key for this trace.
     next_blob_id: u64,
+
+    // TODO: break this up into physical and logical compaction frontiers.
```

Comment:
replace todo with a note saying we can split up the physical and logical compaction parts at a later time